sdk/resourcemanager/azure-resourcemanager-containerregistry/src/main/java/com/azure/resourcemanager/containerregistry/models/Registry.java

```diff
@@ -85,27 +113,53 @@ public interface Registry extends GroupableResource<ContainerRegistryManager, Re
      */
     Mono<SourceUploadDefinition> getBuildSourceUploadUrlAsync();
 
-    /** @return returns entry point to manage container registry webhooks. */
+    /**
+     * Gets entry point to manage container registry webhooks.
+     *
+     * @return returns entry point to manage container registry webhooks.
+     */
     WebhookOperations webhooks();
 
     /**
+     * Gets the state of public network access for the container registry.
+     *
      * @return the state of public network access for the container registry.
      */
     PublicNetworkAccess publicNetworkAccess();
 
-    /** @return whether the container registry can be access from trusted services */
+    /**
+     * Checks whether the container registry can be access from trusted services
+     *
+     * @return whether the container registry can be access from trusted services
+     */
     boolean canAccessFromTrustedServices();
 
-    /** @return the network rule set for the container registry */
+    /**
+     * Gets the network rule set for the container registry.
+     *
+     * @return the network rule set for the container registry
+     */
     NetworkRuleSet networkRuleSet();
 
-    /** @return whether the container registries dedicated data endpoints can be accessed from public network */
+    /**
+     * Checks whether the container registries dedicated data endpoints can be accessed from public network.
+     *
+     * @return whether the container registries dedicated data endpoints can be accessed from public network
+     */
     boolean isDedicatedDataEndpointsEnabled();
 
-    /** @return Whether or not zone redundancy is enabled for this container registry */
+    /**
+     * Checks whether zone redundancy is enabled for this container registry.
+     *
+     * @return Whether zone redundancy is enabled for this container registry
+     */
     boolean isZoneRedundancyEnabled();
 
-    /** @return list of host names that will serve data when isDedicatedDataEndpointsEnabled is true */
+    /**
+     * Gets list of host names that will serve data.
```

Comment:
```suggestion
     * Gets list of host names that will serve data when isDedicatedDataEndpointsEnabled is true.
```

Reply:
Fixed by the [commit#45f4e5a](https://github.com/Azure/azure-sdk-for-java/pull/43517/commits/45f4e5a69f93dddddb9daa8b89a1fed4b85b7ea0)